cmd/readme.md

```diff
@@ -0,0 +1,13 @@
+#Kareljs
+
+##usage
+
+`kareljs compile source.txt -o compiled.kx`
+
+`kareljs run compiled.kx`
+
+##installation
+
+ - Go to /cmd
+ - sudo npm install -g
```

Comment:
nit: hay que poner en diferente fuente si quieres indicar que esos son comandos o cosas literales.

- Go to `cmd/`
- `sudo npm install -g`